hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/RDBStore.java

```diff
@@ -94,6 +98,15 @@ public RDBStore(File dbFile, DBOptions options,
     this.writeOptions = writeOptions;
 
     try {
+      List<TableConfig> columnFamiliesInDb = getColumnFamiliesInExistingDb();
+      List<TableConfig> extraCf = columnFamiliesInDb.stream().filter(
+          cf -> !families.contains(cf)).collect(Collectors.toList());
+      if (CollectionUtils.isNotEmpty(extraCf)) {
+        LOG.info("Found the following extra column families in existing DB : " +
+                "{}", Arrays.toString(extraCf.toArray()));
```

Comment:
Nit: I think `List.toString` is just as good:

```suggestion
                "{}", extraCf);
```